JavaScript1/Week2/homework.md

```diff
@@ -6,200 +6,157 @@
 
-2. Write a JavaScript program which iterates the integers from 1 to 1000. But for multiples of three print "Fizz" instead of the number and for the multiples of five print "Buzz". 
-For numbers which are multiples of both three and five print "FizzBuzz".
+Come up with one feature you think would be helpful for this program.
 
-3. Write a JavaScript program to construct the following pattern, using a nested for loop. This might be bit of a challenge but read on nested for loop.
-    ````js
-        *
-        * *
-        * * *
-        * * * *
-        * * * * *
-        * * * * * *
-    ````
+Optional
+* Lets improve the addActivity, so that we dont need to specify the date, but the function automatically figures out what the date is. Check out this link: https://stackoverflow.com/a/34015511
+* Improve the ShowStatus function by only showing the number of actitivies for that specific day.
+* Create a function for calculating the activity a user has spent the most time on.
+---
 
 
-**3 (Challenge - only do this one once you have finished 1 and 2). Please work closely with your pair we identified in class.**
+## Step 4: Rover the Robot (Optional)
+Go and try out this cool game: [roverjs.com](http://roverjs.com), written by one of the HYF teachers from Amsterdam, Joost Lubach. There are different levels. Solve the chapter up until and with the variables chapter. There might be a small price for completing that!
 
-I would strongly recommend play the game couple of times and adjust the step below if 
-necessary. See also the flow chart we gave you.
+## Step 5: Hand in Homework:
+Steps:
+* Use the hyf-homework repo and add your homework files in the javascript/javascript1/week2 folder
+* If you dont have the repo, fork it from here: https://github.com/HackYourFuture-CPH/hyf-homework
+* Make your commits for the homework. Remember to seperate the code into meaningful commits!
+* Push the changes to github
+* To finish the homework:
+  * Fill in the form for the relevant week. The link to the form can be found at the top of your classes slack channel. 
+  * Share the github link to your classes slack channel. 
+---
 
-1. Create an object of ``card``
-2. Create an object of ``player`` with the following : ``name``, ``age``, ``winner``
-3. Create an object for ``game`` with the following variables: array of four players and four arrays of cards.
-4. Each array belongs to one player.
-5. Create a function to distribute the cards to the four arrays.
-6. Take the next player in the array. 
-7. Find cards with highest frequency.
-8. Go to the next player and see if the player has the card you want
-9. If it does continue and go to 7. If not go to 6.
-10. If at any point there is one happy family in one of the array change the winner key to "true"
-declare winner.
+And thats it your are done 🎉
```

Comment:
That was AWESOME!! Another amazing homework. WHO WOULDN'T WANT TO DO THIS?!?!! I wonder if people will understand the Smarties pun 🤔 https://www.quiltsmart.com/media/ecom/prodsm/WEB%20bb%20cover.jpg

Reply:
Ah, that's funny with the actual smarties cover :) 
Maybe some will, if they dont get it, the name still sounds kind of techy, so it does not matter i guess.